w3w-autosuggest/admin/index.php

```diff
@@ -0,0 +1 @@
+<?php // Silence is golden
```

Comment:
...Is this meant to be here? 😅

Reply:
Yes is actually a PHP/WordPress thing funnily enough.